test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -0,0 +1,245 @@
+using Tizen.NUI.Scene3D;
+using System.Collections.Generic;
+
+class Scene3DSample : NUIApplication
+{
+    static string IMAGE_DIR = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "image/";
+    static string MODEL_DIR = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "model/";
+
+    Window mWindow;
+    Vector2 mWindowSize;
+
+    SceneView mSceneView;
+    Model     mModel;
+
+    uint currentCameraIndex = 0u;
+    const int cameraAnimationDurationMilliSeconds = 2000; // milliSeconds
+
+    Animation mModelRotateAnimation;
+    const int modelRotateAnimationDurationMilliSeconds = 10000; // milliSeconds
+
+    protected void CreateSceneView()
+    {
+        mSceneView = new SceneView()
+        {
+            SizeWidth = mWindowSize.Width,
+            SizeHeight = mWindowSize.Height,
+            PositionX = 0.0f,
+            PositionY = 0.0f,
+            PivotPoint = PivotPoint.TopLeft,
+            ParentOrigin = ParentOrigin.TopLeft,
+            PositionUsesPivotPoint = true,
+        };
+
+        SetupSceneViewCamera(mSceneView);
+
+        mWindow.Add(mSceneView);
+    }
+    private void SetupSceneViewCamera(SceneView sceneView)
+    {
+        // Default camera setting
+        // Note : SceneView always have 1 default camera.
+        Tizen.NUI.Scene3D.Camera defaultCamera = sceneView.GetCamera(0u);
+
+        defaultCamera.PositionX = 0.0f;
+        defaultCamera.PositionY = 0.0f;
+        defaultCamera.PositionZ = 5.55f;
+        defaultCamera.NearPlaneDistance = 0.5f;
+        defaultCamera.FarPlaneDistance = 50.0f;
+        //defaultCamera.Orientation = new Rotation(new Radian(new Degree(180.0f)), Vector3.YAxis);
+        defaultCamera.FieldOfView = new Radian(new Degree(45.0f));
+        defaultCamera.OrthographicSize = 2.7f;
+
+        // Additional camera setting (top view camera).
+        Tizen.NUI.Scene3D.Camera camera = new Tizen.NUI.Scene3D.Camera()
+        {
+            PositionX = 0.0f,
+            PositionY = -3.95f,
+            PositionZ = 0.0f,
+            NearPlaneDistance = 0.5f,
+            FarPlaneDistance = 50.0f,
+            // Rotate by XAxis first, and then rotate by YAxis
+            Orientation = new Rotation(new Radian(new Degree(90.0f)), Vector3.YAxis) *
+                            new Rotation(new Radian(new Degree(-90.0f)), Vector3.XAxis),
+            FieldOfView = new Radian(new Degree(70.0f)),
+            OrthographicSize = 2.7f,
+        };
+        sceneView.AddCamera(camera);
+    }
+
+    protected void CreateModel(string modelUrl)
+    {
+        // Release old one.
+        if(mModel != null)
+        {
+            mModel.Unparent();
+            mModel.Dispose();
+        }
+
+        mModel = new Model(MODEL_DIR + modelUrl)
+        {
+            Name = modelUrl,
+        };
+        mModel.ResourcesLoaded += (s, e) =>
+        {
+            Model model = s as Model;
+
+            // You can play animation if it has.
```

Comment:
It would be better :
```suggestion
            // You can play animation if the animation exists.
```

Reply:
Done.